### PR TITLE
[Release 3.1.0] Update change log and module manifest files

### PR DESCRIPTION
**Pull request by cormacpayne**
Used `tools\UpdateChangeLog.ps1` file to update change logs and module manifest files for each service.

### CODE REVIEW

**Comment by cormacpayne** [comment]
@markcowl 

On-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1285/

File: src/ResourceManager/AzureStackAdmin/AzureRM.AzureStackAdmin.psd1 [hunk 1]
```diff
@@ -33,60 +36,81 @@ Description = 'Microsoft Azure Stack Administration Module'
...
 CLRVersion = '4.0'
 
-# Processor architecture (None, X86, Amd64, IA64) required by this module
-ProcessorArchitecture = ''
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @( @{ ModuleName = 'AzureRM.Profile'; ModuleVersion = '2.3.0'})
+RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '2.0.1'; })
```
Comment (markcowl) [inline src/ResourceManager/AzureStackAdmin/AzureRM.AzureStackAdmin.psd1]: RequiredModules change

File: src/ResourceManager/AzureStackAdmin/AzureRM.AzureStackAdmin.psd1 [hunk 2]
```diff
@@ -33,60 +36,81 @@ Description = 'Microsoft Azure Stack Administration Module'
...
 # FileList = @()
 
-# Private data to pass to the module specified in ModuleToProcess
+# Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
 PrivateData = @{
 
     PSData = @{
 
         # Tags applied to this module. These help with module discovery in online galleries.
-        Tags = @("Azure", "ResourceManager", "ARM", "AzureStack")
```
Comment (markcowl) [inline src/ResourceManager/AzureStackAdmin/AzureRM.AzureStackAdmin.psd1]: Tags change -looks liek this was not part of your build

File: src/ResourceManager/AzureStackStorage/AzureRM.AzureStackStorage.psd1 [hunk 3]
```diff
@@ -1,94 +1,117 @@
...
+# Generated on: 10/31/2016
+#
+
+@{
+
+# Script module or binary module file associated with this manifest.
+# RootModule = ''
+
+# Version number of this module.
+ModuleVersion = '0.9.91'
```
Comment (markcowl) [inline src/ResourceManager/AzureStackStorage/AzureRM.AzureStackStorage.psd1]: wrong version

File: src/ResourceManager/AzureStackStorage/AzureRM.AzureStackStorage.psd1 [hunk 4]
```diff
@@ -1,94 +1,117 @@
...
+DotNetFrameworkVersion = '4.0'
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+CLRVersion = '4.0'
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '2.0.1'; })
```
Comment (markcowl) [inline src/ResourceManager/AzureStackStorage/AzureRM.AzureStackStorage.psd1]: wrong version, looks like this was not in your build

File: src/ResourceManager/AzureStackStorage/AzureRM.AzureStackStorage.psd1 [hunk 5]
```diff
@@ -1,94 +1,117 @@
...
+# FileList = @()
+
+# Private data to pass to the module specified in RootModule/ModuleToProcess. This may also contain a PSData hashtable with additional module metadata used by PowerShell.
 PrivateData = @{
 
     PSData = @{
 
         # Tags applied to this module. These help with module discovery in online galleries.
-        Tags = @("Azure", "ResourceManager", "ARM", "AzureStack")
+        # Tags = @()
```
Comment (markcowl) [inline src/ResourceManager/AzureStackStorage/AzureRM.AzureStackStorage.psd1]: tags removed

File: src/ResourceManager/AzureStackStorage/ChangeLog.md [hunk 6]
```diff
@@ -17,4 +17,6 @@
     * Overview of change #1
         - Additional information about change #1
 -->
-## Current Release
\ No newline at end of file
+## Current Release
+
+## Version 0.9.91
```
Comment (markcowl) [inline src/ResourceManager/AzureStackStorage/ChangeLog.md]: wrong version

File: src/ResourceManager/AzureStackAdmin/ChangeLog.md [hunk 7]
```diff
@@ -17,4 +17,6 @@
     * Overview of change #1
         - Additional information about change #1
 -->
-## Current Release
\ No newline at end of file
+## Current Release
+
+## Version 0.9.10
```
Comment (markcowl) [inline src/ResourceManager/AzureStackAdmin/ChangeLog.md]: wrong version

**Comment by cormacpayne** [comment]
On-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1286/

**Comment by cormacpayne** [comment]
On-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1287/

Signing: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-sign/672/

**Pull request merged**